python/ray/autoscaler/aws/example.yaml

```diff
@@ -0,0 +1,65 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: default
+
+# The minimum number of workers nodes to launch in addition to the head
+# node. This number should be >= 0.
+min_workers: 0
+
+# The maximum number of workers nodes to launch in addition to the head
+# node. This takes precedence over min_workers.
+max_workers: 2
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-east-1
```

Comment:
how about making the default `us-west-2`?

Reply:
We can later, but right now the testing AMI i'm using is in us-east-1